### PR TITLE
cod3.d: fix whitespace

### PR DESCRIPTION
**Pull request by WalterBright**
Whitespace, just whitespace

### CODE REVIEW

File: src/dmd/backend/cod3.d [hunk 1]
```diff
@@ -419,7 +419,7 @@ else
  *      seg = segment to write alignment bytes to
  *      nbytes = number of alignment bytes to write
  */
-void cod3_align_bytes(int seg, size_t nbytes)
+void cod3_align_bytes(int seg, uint nbytes)
```
Comment (Geod24) [inline src/dmd/backend/cod3.d]: That's not whitespace

File: src/dmd/backend/cod3.d [hunk 2]
```diff
@@ -2438,29 +2446,29 @@ else
...
+                elem *e = el_long(TYnptr, 0);
+                e.Eoper = OPgot;
+                regm_t retregs = mBX;
+                codelem(cdb,e,&retregs,false);
+                el_free(e);
+            }
         }
     }
 }
-}
```
Comment (Geod24) [inline src/dmd/backend/cod3.d]: Not whitespace either, but it's hard to see which opening brace it's matching ?

File: src/dmd/backend/cod3.d [hunk 3]
```diff
@@ -1978,46 +1980,48 @@ int jmpopcode(elem *e)
...
+    else
         zero = 0;
 
-  tym = e.EV.E1.Ety;
-  if (tyfloating(tym))
-  {
+    tym = e.EV.E1.Ety;
+    if (tyfloating(tym))
+    {
 static if (1)
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: I would indent the static if.

File: src/dmd/backend/cod3.d [hunk 4]
```diff
@@ -2066,39 +2070,39 @@ else
...
         i = 1;
-  else if (tyintegral(tym) || typtr(tym))
+    else if (tyintegral(tym) || typtr(tym))
         i = 0;
-  else
-  {
+    else
+    {
         debug
         elem_print(e);
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: Indent.

File: src/dmd/backend/cod3.d [hunk 5]
```diff
@@ -2066,39 +2070,39 @@ else
...
         jp = (jp == JA) ? JC : JNC;
-  }
+    }
 
 L1:
-  debug
-  if ((jp & 0xF0) != 0x70)
-  {
+    debug
+    if ((jp & 0xF0) != 0x70)
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: Indent.

File: src/dmd/backend/cod3.d [hunk 6]
```diff
@@ -5587,30 +5740,33 @@ debug
...
             }
         }
-  }
+    }
 
-  debug
-  if (debugc) {
-      printf("-pinholeopt(%p)\n",cstart);
+    debug
+    if (debugc)
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: Indent.

File: src/dmd/backend/cod3.d [hunk 7]
```diff
@@ -5587,30 +5740,33 @@ debug
...
             code_print(c);
-  }
-
+    }
 }
 
+
 debug
 {
 private void pinholeopt_unittest()
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: Indent.

File: src/dmd/backend/cod3.d (hunk 2 above)
Comment (WalterBright) [reply src/dmd/backend/cod3.d]: That's just github diff getting confused with the change in indentation.

File: src/dmd/backend/cod3.d (hunk 4 above)
Comment (WalterBright) [reply src/dmd/backend/cod3.d]: I've been trying different styles for that, not sure which is best.

File: src/dmd/backend/cod3.d [hunk 8]
```diff
@@ -6707,252 +6896,267 @@ static if (TARGET_OSX)
...
+        case FLcsdata:
+        case FLfardata:
+            //symbol_print(uev.Vsym);
+
+            // NOTE: In ELFOBJ all symbol refs have been tagged FLextern
+            // strings and statics are treated like offsets from a
+            // un-named external with is the start of .rodata or .data
+        case FLextern:                      /* external data symbol         */
+        case FLtlsdata:
+    static if (TARGET_LINUX || TARGET_FREEBSD || TARGET_OPENBSD || TARGET_DRAGONFLYBSD || TARGET_SOLARIS)
```
[the remark this answers is not in the document]
Comment (WalterBright) [reply src/dmd/backend/cod3.d]: No good options there.

[the remark this answers is not in the document]
Comment (jacob-carlborg) [reply src/dmd/backend/cod3.d]: Perhaps some future refactoring could avoid the need for this.

File: src/dmd/backend/cod3.d (hunk 1 above)
Comment (WalterBright) [reply src/dmd/backend/cod3.d]: fixed

**Pull request merged**